docs/platform/developing/flashing-rpi.md

```diff
@@ -22,6 +22,10 @@ IoT Setup Manager supports the following operating systems:
 -   Windows 10
 -   Ubuntu 18.04 LTS and later
 
+## SD card set up
+
+The SD card must be formatted before flashing image into it.
```

Comment:
```suggestion
The SD card must be formatted before flashing the image into it.
```

Reply:
Thanks for you review. This comment is fixed.